### ISSUE TITLE
Some confusion about local attention 

### ISSUE THREAD

**Issue by ccuiyuhan** [noperoc/pytorch-attention-Banhdanau-Luong#1]
`      
       sigmoid_inp = self.omega_v(F.tanh(self.omega_w(s))).squeeze(1)

        S = self.s_len - 2*self.D - 1

        ps = S * F.sigmoid(sigmoid_inp)

        pt = ps + self.D

        batch_size = enc_output.shape[0]

        enc_output_ = torch.zeros(1, 2*self.D+1, args.hidden_size, device=device)`

Excuse me, i have some questions about the local attention, in the paper, S is the source sentence length, why S = self.s_len - 2*self.D - 1 in your code, may be S = self.s_len? Also, in the paper, pt = S · sigmoid(v * tanh(W*h)), why pt = ps + self.D here. i think, ps (ps = S * F.sigmoid(sigmoid_inp))is pt in your code.
 I don't know if I am right or not.Hope the reply! Thanks!
